### PR TITLE
Moderation access

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by Trott** [comment]
Can you please run this (and all doc contributions) through a spelling/grammar checker?

**Comment by mhdawson** [comment]
One note is that we are still moving some repos over to nodejs-private, and we should probably wait until that is done to land this (if we reach consensus).  I don't think that moving them over should take more than a few days.

File: GitHub-Org-Management-Policy.md [hunk 1]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
 
-TSC members and the Chair of the Community Committee are the only individuals granted
-Owner permissions within the Node.js GitHub Organization.
+As with other resources within the community, Org ownership will be granted in order
+to optimize for the conflicting requirements of:
+
+* Limiting access to reduce risk
+* Enabling individuals to move community work forward without undue delay
+
+When possible we should use automation and tools to reduce the breadth of access
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: add comma after `possible`

**Comment by mhdawson** [comment]
@Trott  do you have a good suggestion for one that you use when generating md files ?

File: GitHub-Org-Management-Policy.md (hunk 1 above)
Comment (Trott) [reply GitHub-Org-Management-Policy.md]: Probably don't want to use `we` here or on line 27.

File: GitHub-Org-Management-Policy.md [hunk 2]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+that needs to be provide in order to enable individuals to move community work foward.
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: typo: `be provide` -> `be provided`

Comment (Trott) [reply GitHub-Org-Management-Policy.md]: `forward` is misspelled

File: GitHub-Org-Management-Policy.md [hunk 3]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+As these tools are developed and put in place we will reduce the groups to which
+org ownership is granted.
+
+At the current time the following groups are granted access in order for them
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: add a comma after `time`

File: GitHub-Org-Management-Policy.md [hunk 4]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+to facilitate community work.
+
+* TSC members
+* Chair of the Community Comittee
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: `Committee` is misspelled

File: GitHub-Org-Management-Policy.md [hunk 5]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+* Moderation team members. - It is expected that moderation team members will limit
+their use of the access granted to that requied to carry out moderation across the
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: `required` is misspelled

**Comment by Trott** [comment]
> @Trott do you have a good suggestion for one that you use when generating md files ?

I usually use the free version of Grammarly, although I don't know how well it does with raw markdown. I have a Markdown editor, so I copy/paste the rendered doc into Grammarly rather than the raw markdown. But I'd guess it's fine, especially for a doc like this. It can generate a lot of false positives, but I think that's pretty much true of all the grammar tools.

**Comment by mhdawson** [comment]
Pushed commit addressing initial comments.

File: GitHub-Org-Management-Policy.md [hunk 6]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
 
 ### Owners
 
-TSC members and the Chair of the Community Committee are the only individuals granted
-Owner permissions within the Node.js GitHub Organization.
+As with other resources within the community, Org ownership will be
+granted in order to optimize for the conflicting requirements of:
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: We don't use "Org ownership" in the doc. That phrase is pretty colloquial. Stick to the terminology elsewhere: Owner permissions.

"As with other resources within the community..." is filler. Take it out.

So, all told, maybe something like this?:

> Whether to grant of Owner permissions is determined by optimizing for the following conflicting requirements:

(As an alternative, I'd also be all for taking out all of this explanatory material and just identifying the people that have Owner permissions.)

File: GitHub-Org-Management-Policy.md [hunk 7]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+
+* Limiting access to reduce risk
+* Enabling individuals to move community work forward without undue delay
+
+When possible, automation and tools should be used to reduce the breadth of
+access that needs to be provided in order to enable individuals to move
+community work forward.  As these tools are developed and put in place we will
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: How about this instead?:

> As these tools are developed and put in place, the groups to which Owner permissions are granted will be reduced.

File: GitHub-Org-Management-Policy.md [hunk 8]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+reduce the groups to which org ownership is granted.
+
+At the current time, the following groups are granted access in order for them
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: Nit: "At the current time" is superfluous. Take it out.

Comment (Trott) [reply GitHub-Org-Management-Policy.md]: Also consider removing "in order for them to facilitate community work.". The sentence should end with a colon as well. So all told, maybe this?:

> The following groups are granted Ownership permissions:

File: GitHub-Org-Management-Policy.md [hunk 9]
```diff
@@ -16,8 +16,25 @@ Committee members have write permissions to the Node.js admin repository.
...
+to facilitate community work.
+
+* TSC members
+* Chair of the Community Committee
+* Moderation team members. - It is expected that moderation team members
```
Comment (Trott) [inline GitHub-Org-Management-Policy.md]: Nit: remove the dash/hyphen/whatever.

Comment (Trott) [reply GitHub-Org-Management-Policy.md]: Nit: Get rid of "It is expact that".

Comment (Trott) [reply GitHub-Org-Management-Policy.md]: Generally, I think we've been styling it captialized in the docs: `Moderation Team`

**Comment by Trott** [comment]
The TL;DR of this seems to be "Moderation Team gets Owner permissions until there's some other way for them to do their work, which will hopefully be a bot at some point in the not-too-distant future." 

+1 from me!

**Comment by jasnell** [comment]
Given the objections that @rvagg has raised on this, I'd like for this to to on the tsc-agenda for discussion before it lands. I'm not -1'ing the actual change, I just want it to be discussed on the call.

**Comment by mhdawson** [comment]
Was discussed in last TSC meeting, no objections to moving forward.  James said we could dismiss his objection.

**Comment by jasnell** [comment]
Also from the TSC meeting: Everyone with owner access needs to have 2FA enabled on their github account. That needs to be confirmed before assigning the owner permission. Also, efforts to pursue automation options need to move forward so that assigning owner permissions to individuals outside the TSC and CommComm chairs no longer becomes necessary. The rationale there is strictly limited to general security concerns -- that is, the higher the number of users with owner permissions, the higher the risk of something bad happening should one of those accounts be compromised. So granting owner permissions to the moderation team should be considered a temporary action *until* an automated solution can be developed.

**Comment by mhdawson** [comment]
@Trott took the action to validate 2FA before adding owner access.

**Comment by jasnell** [comment]
> took the action to validate 2FA before adding owner access

Yep... for existing mod team members... it needs to be confirmed for *new* members before assigning permissions.

**Comment by mhdawson** [comment]
Landed as 9cf274c532ca22bf91e43d6d87af1e6aac59fd0f

**Pull request closed without merging**